lighthouse-core/config/default-config.js

```diff
@@ -559,6 +560,7 @@ const defaultConfig = {
         {id: 'viewport', weight: 2, group: 'pwa-optimized'},
         {id: 'without-javascript', weight: 1, group: 'pwa-optimized'},
         {id: 'apple-touch-icon', weight: 1, group: 'pwa-optimized'},
+        {id: 'maskable-icon', weight: 1, group: 'pwa-optimized'},
```

Comment:
@paulirish is the PWA council still a thing that we need to be thinking about and consulting when making changes that affect badging? or at this point am I carrying on about olden times for no good reason.

Reply:
the conclusion to in person discussion on this was that we do need to consult the council when things affect "fast and reliable" but "pwa optimized" we control and is fine to modify